src/devices/lcec_epocat.c

```diff
@@ -615,110 +703,135 @@ static void lcec_epocat_write(struct lcec_slave *slave, long period) {
   EC_WRITE_BIT(&pd[hal_data->off_ENABLE_5V], 3, *(hal_data->ENABLE_5V_3));
   EC_WRITE_BIT(&pd[hal_data->off_ENABLE_5V], 4, *(hal_data->ENABLE_5V_4));
 
-  fValue = (((*(hal_data->dac_00_value) * 1000) / hal_data->enc_00_scale) / 16) * hal_data->dac_00_scale;
-  if (fValue > 65534) {
-    fValue = 65534;
+  fValue = (((*(hal_data->dac_00_value) * 1000) / hal_data->enc_00_scale)) * hal_data->dac_00_scale;
+  /*if (fValue > 65534)
```

Comment:
I'm not sure why this is commented out, either.  Is this because the underlying variable changed from 16 bits to 32 bits?  I'd recommend keeping the range checks, because it's still possible to overflow 32-bit ints.  I *think* most (all?) of the vars in this are floats, so you don't need to worry about overflow in intermediate calculations, but it'd still be good to not overflow the registers on the device.

Reply:
I commented out because now the range is the full 4 bytes value 0 to 4.194.304

Reply:
Please remove the commented lines.